common/display/displayplanemanager.cpp

```diff
@@ -111,6 +111,9 @@ std::tuple<bool, DisplayPlaneStateList> DisplayPlaneManager::ValidateLayers(
   auto layer_begin = layers.begin();
   auto layer_end = layers.end();
   bool render_layers = false;
+  // Below flag takes care of condition,when the cursor layer is assigned the
+  // lower zorder
+  bool cursor_plane_enable = true;
```

Comment:
https://github.com/01org/IA-Hardware-Composer/blob/master/os/android/drmhwctwo.cpp#L443

Can u check we are having the right z order here, 
https://github.com/01org/IA-Hardware-Composer/blob/master/public/nativedisplay.h#L86  expectation is that we have layers in increasing z order (0 = primary  etc)